pragma-deployer/pyproject.toml

```diff
@@ -13,7 +13,6 @@ register-publishers = 'pragma_deployer.register_publishers:cli_entrypoint'
 deploy-summary-stats = 'pragma_deployer.deploy_summary_stats:cli_entrypoint'
 deploy-randomness = 'pragma_deployer.deploy_randomness:cli_entrypoint'
 deploy-randomness-example = 'pragma_deployer.deploy_randomness_example:cli_entrypoint'
-deploy-lp-pricer = 'pragma_deployer.deploy_lp_pricer:cli_entrypoint'
```

Comment:
?

Reply:
We don't have any lp_pricer deployment script - we removed it but kept this line